azure/ConsiderationsForServiceDesign.md

```diff
@@ -90,6 +97,13 @@ Before releasing your API plan to invest significant design effort, get customer
 
 :ballot_box_with_check: **YOU SHOULD**  capture what you have learned during the preview stage and share these findings with your team and with the API Stewardship Board.
 
+## Communicate Deprecations
+As your service evolves over time, it will be natural that you want to remove operations that are no longer needed. For example, additional requirements or new capability in your service, may have resulted in a new operation that, effectively, replaces an old one.
```

Comment:
I wouldn't say it's "natural."  I want people to think about public APIs like marriage - it's forever unless you're so unhappy supporting it you're willing to pay for a very expensive breaking change lawyer.

```suggestion
As your service evolves over time, you may need to remove operations that are no longer supportable. For example, additional requirements or a new capability in your service may result in a new operation that effectively replaces an old one.
```